Gemfile

```diff
@@ -53,7 +53,7 @@ gem 'avalon-about', git: 'https://github.com/avalonmediasystem/avalon-about.git'
 #gem 'bootstrap-sass', '< 3.4.1' # Pin to less than 3.4.1 due to change in behavior with popovers
 gem 'bootstrap-toggle-rails'
 gem 'bootstrap_form'
-gem 'iiif_manifest', '>= 1.4.0'
+gem 'iiif_manifest', git: 'https://github.com/samvera/iiif_manifest.git', branch: 'canvas-homepage'
```

Comment:
The changes from that branch have been merged into iiif_manifest so should we point to main for this release? Or we could bump the iiif_manifest version to 1.4.1 (or would semantic versioning require a bump to 1.5.0 since it is adding functionality?)

Reply:
Good catch!  How about we cut a `1.5.0` since it does add functionality?  I made a PR: https://github.com/samvera/iiif_manifest/pull/111